roles/neutron-control/tasks/monitoring.yml

```diff
@@ -22,3 +22,10 @@
                        args='-S neutron --scheme {{ monitoring.graphite.host_prefix }}'
   notify: restart sensu-client
 
+- name: lbaas sla metrics
+  sensu_metrics_check: name=lbaas-sla-metrics plugin=metrics-os-api.py
+                       args='-S lbaas --scheme {{ monitoring.graphite.host_prefix }}'
+  when: (neutron.lbaas.enable == "smart" and
+	+ groups['controller'][0] not in groups['compute']) or
+	+ neutron.lbaas.enable|bool
```

Comment:
Your copy/paste included the literal `+` signs from wherever you got the content from. This broke the run.